lib/blockingproxy.ts

```diff
@@ -216,7 +234,19 @@ export class BlockingProxy {
                 response.writeHead(seleniumResponse.statusCode, seleniumResponse.headers);
                 seleniumResponse.pipe(response);
               });
-          originalRequest.pipe(seleniumRequest);
+          let reqData = '';
+          originalRequest.on('data', (d) => {
```

Comment:
I added error handlers - I'm planning to refactor this bit and add some unit tests specifically for errors soon.